### ISSUE TITLE
Require more details on the Dataset used

### ISSUE THREAD

**Issue by Bloodrose91** [l294265421/NovelNet#2]
Hi
I am having difficulty understanding the raw data I got. What does each column represent?
Column names: Index(['SessionId', 'TimeStr', 'ItemId', 'DataType', 'Attributes']

And in the Attributes column, what do each of these mean?
{"book_id": "1138750515", "bookname": "\u9752\u4e91\u4ed9\u65cf", "category": "\u4ed9\u4fa0", "subtype": "\u53e4\u5178\u4ed9\u4fa0", "author": "\u5357\u6781\u80d6\u718a", "start_time": "2021-11-19 20:11:18", "end_time": "2021-11-19 20:11:33", "expose": 0, "click": 1, "intro": 1, "read": 1, "real_read": 1, "collect": 0, "read_duration": 4.0, "set": "training"}

**Comment by l294265421** [comment]
@Bloodrose91 Thanks for your interest in our work. The explanations are as follows:
- In the data, one row represents a interaction between a user and a book, such as click.
- SessionId: the interactions with the same session id are continuous. The definition of session is the same as that in the Session-based Recommendation task.
- ItemId: the unique id of the book in the dataset.
- TimeStr: the start time of the session in str format.
- DataType has following values:
  - 1: the interaction is only used as features to predict future interactions
  - 2: the interaction belongs to training set, , that is, this interaction will be used as the label of an instance in traning set.
  - 3: the interaction belongs to validation set.
  - 4: the interaction belongs to test set.
  - 5: the interaction is not used.
- Attributes: extra information of the book and the interaction whose id is ItemId, such as:
  - bookname
  - category: book category, eg. Romance
  - subtype: more fine-grained book category
  - author
  - start_time: start time of the interaction
  - end_time: end time of the interaction
  - expose: 0 or 1, indicating whether this is an expose interaction.
  - click: 0 or 1, indicating whether this is an clickinteraction.
  - intro: 0 or 1, indicating whether this is an intro interaction.
  - read: 0 or 1, indicating whether this is an read interaction.
  - collect: 0 or 1, indicating whether this is an collect interaction.
  - read_duration: the duration of the interaction
  - set: dataset type, training, validation, test

**Comment by Bloodrose91** [comment]
@l294265421 l294265421

Thank you for the information. Just to clarify, SessionId is a UserID? 
as it is unique to 1 Session = 1 user ?

**Comment by Bloodrose91** [comment]
Hi 
Sorry to disturb you once again, but would like to clarify if the variable read_duration: the duration of the interaction
 is it in seconds, minutes, or hours?

**Comment by l294265421** [comment]
> Hi Sorry to disturb you once again, but would like to clarify if the variable read_duration: the duration of the interaction is it in seconds, minutes, or hours?

seconds
